aiida/orm/querybuilder.py

```diff
@@ -1262,7 +1262,7 @@ def _join_descendants_recursive(self, joined_entity, entity_to_join, isouterjoin
         """
 
         self._check_dbentities((joined_entity, self._impl.Node), (entity_to_join, self._impl.Node),
-                               'descendant_of_beta')
+                               'with_ancestors_beta')
```

Comment:
Is this `_beta` version still implemented? Probably @lekah can answer this

Reply:
No, what we're doing now (transitive closure on the fly) used to be called _beta. This string passed is just for the exception, and can be changed without a problem (i.e. remove _beta in the string).
My suggest: without a new issue within this pull request, if noone objects.

Reply:
Good for me!

Reply:
Made these changes.